Source/WebCore/loader/CrossOriginAccessControl.h

```diff
@@ -58,7 +58,7 @@ void updateRequestReferrer(ResourceRequest&, ReferrerPolicy, const String&);
     
 WEBCORE_EXPORT void updateRequestForAccessControl(ResourceRequest&, SecurityOrigin&, StoredCredentialsPolicy);
 
-WEBCORE_EXPORT ResourceRequest createAccessControlPreflightRequest(const ResourceRequest&, SecurityOrigin&, const String&);
+WEBCORE_EXPORT ResourceRequest createAccessControlPreflightRequest(const ResourceRequest&, SecurityOrigin&, const String&, bool);
```

Comment:
Should we use an enum for this bool?

`enum class FetchMetadata : bool` ?

```
FetchMetadata::Exclude
FetchMetadata::Include
```

Reply:
On second thought, I guess this is just about an enablement flag, so maybe just leave it bool.

Reply:
We should still *NOT* omit the parameter name, since it is not obvious what this boolean is for otherwise.